packages/components/src/List/DisplayLarge/DisplayLarge.component.js

```diff
@@ -170,8 +171,11 @@ function DisplayLarge(props) {
 		theme.container,
 		'tc-list-display',
 	);
+	const hasItems = items && items.length !== 0;
```

Comment:
Test on `items` truthiness is not necesary as we have defaultProps.
So you can remove this const and test directly `items.length` in the render()